app/components/primer/alpha/action_menu/action_menu_element.ts

```diff
@@ -105,8 +113,8 @@ export class ActionMenuElement extends HTMLElement {
 
     if (!this.popoverElement?.matches(':popover-open')) return
 
-    if (event.type === 'focusout' && !this.contains((event as FocusEvent).relatedTarget as Node)) {
-      this.popoverElement?.hidePopover()
```

Comment:
Why has this functionality been removed? I think we need this

Reply:
Ah ok I was afraid of that. What does it do? None of our tests failed when I removed it. Also, it definitely causes the issue linked in the description. The popover disappears after the content loads.

Reply:
Hmm ok interesting. Let me add a test for that and see if there's a way to distinguish between the two cases, i.e. opening another menu/dialog and loading deferred content.

Reply:
Ok, I added a test for this that renders two menus and it passed without the original `focusout` code.

Reply:
Also just added an assertion ensuring the menu closes when a dialog opens.